tests/foreman/ui/test_org.py

```diff
@@ -609,22 +618,23 @@ def test_add_location(self, location_name):
 
         """
         strategy, value = common_locators['entity_deselect']
-        org_name = gen_string('alpha')
-        location = entities.Location(name=location_name).create()
-        self.assertEqual(location.name, location_name)
         with Session(self.browser) as session:
```

Comment:
Same comment as before about using one single organization and adding entity to it and asserting its presence as well as increase of entities associated to the organization. Not a blocker for this PR.